PizzaPyWebApp/templates/partials/landing/hero.html

```diff
@@ -1,5 +1,7 @@
 <div class="relative flex flex-col items-center justify-center w-full min-h-screen">
-    <div class="flex flex-col items-center gap-10 p-6 sm:gap-20 md:p-28 dark:bg-gray-900">
+    <img src="https://via.placeholder.com/1500x800" alt="Background Image"
```

Comment:
This can be removed because the Hero will not have a static background image. Just the generic background color.